pkg/platform/packet/packet.go

```diff
@@ -237,6 +252,25 @@ func (c *config) terraformSmartApply(ex *terraform.Executor, dnsProvider dns.DNS
 	return ex.Apply()
 }
 
+// terraformCreateReservations creates nodes that use a specific hardware
+// reservation ID
+func (c *config) terraformCreateReservations(ex *terraform.Executor) error {
+	arguments := []string{"apply", "-auto-approve"}
+
+	// Create workers that use specific UUIDS as hardware reservation
```

Comment:
```suggestion
	// Create workers that use specific UUIDS as hardware reservation.
```

Reply:
Following the [link you posted](https://github.com/golangci/golangci-lint/pull/993) I see that [golang has documented the style](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences) for *documentation* to have a period at the end. But haven't seen anything on *code* comments.

Did you find any place where it is documented? I had a quick look at go source code and it seems they use the same style for code comments too, except when comment is on the same line as code (i.e `a := "hola" // some comment with no period at the end`)

In any case, I think is fine to use always the same style. Just curious if you found it documented in some other place